sdk-cpp/include/do_exceptions.h

```diff
@@ -4,10 +4,13 @@
 #ifndef _DELIVERY_OPTIMIZATION_DO_EXCEPTIONS_H
```

Comment:
>_DELIVERY_OPTIMIZATION_DO_EXCEPTIONS_H [](http://example.com/codeflow?start=8&length=38)

Considering renaming this file now that exceptions can be turned off, to not confuse users. do_errors.h?